gwswlib/tests/test_threedi.py

```diff
@@ -36,6 +36,25 @@ def test_get_mapping_value_right(caplog):
     assert shape == "rnd"
 
 
+def test_check_if_element_is_created_twice(caplog):
+    checked_element = "knp6"
+    created_elements = [
+        {
+            "code": "knp1",
+            "initial_waterlevel": None,
+            "geom": (241330.836, 483540.234, 28992),
+        },
+        {
+            "code": "knp6",
+            "initial_waterlevel": None,
+            "geom": (241463.858, 483356.833, 28992),
+        },
+    ]
+    element_type = "Connection node"
+    check_if_element_is_created_twice(checked_element, created_elements, element_type)
```

Comment:
Hm.... Ik *weet* dat je qua programmaontwerp altijd door wilt gaan met importeren, ongeacht fouten in de inputdata.
Maar ik blijf het lastig vinden dat een methode die "check" heet niet echt iets merkbaars doet (behalve wat loggen).

Het klopt volledig met je ontwerp, maar voor een reguliere programmeur is het een beetje wennen :-)

Wat dit betekent is dat je op de naamgeving moet letten. Iets met "check" in de naam moet dan **overal** alleen maar een beetje controleren en eventueel wat loggen.
Als er een methode is die ook een exceptie kan raisen moet je die een aggressievere naam geven zoals "validate" of "verify" of "assure".